There is currently the following mypy failure:

```shell
sympy/physics/mechanics/_actuator.py:392: error: Read-only property cannot override read-write property  [misc]
```

This is occurring because `ForceActuator` implements its `force` attribute (which is also a parameter to its `__init__`) as a `property` with a setter, while `LinearSpring`, which subclasses `ForceActuator`, implements its `force` attribute (which is _not_ a parameter to its `__init__`) as a `property` without a setter. I.e., `ForceActuator`'s `force` is mutable, whereas `LinearSpring`'s `force` is computed.

Options for fixing this:
1. Refactor all actuator classes to make their attributes immutable (by deleting the property setters and refactoring the type checking logic to their `__init__` methods). This is my preferred option.
2. Remove the inheritance relationship between `LinearSpring` and `ForceActuator`. While one could argue that the current implementation of changing `__init__` parameters between subclasses is flakey design, I'd argue that the usefulness of the inheritance hierarchy for handling logic where actuator classes are used in the rest of `sympy.physics.mechanics` trumps it in this case.

--- a/sympy/physics/mechanics/_actuator.py
+++ b/sympy/physics/mechanics/_actuator.py
@@ -33,7 +33,11 @@ if TYPE_CHECKING:
     from sympy.physics.mechanics.loads import LoadBase
 
 
-__all__ = ['ForceActuator']
+__all__ = [
+    'ForceActuator',
+    'LinearSpring',
+    'TorqueActuator',
+]
 
 
 class ActuatorBase(ABC):

--- a/sympy/physics/mechanics/_actuator.py
+++ b/sympy/physics/mechanics/_actuator.py
@@ -155,17 +155,6 @@ class ForceActuator(ActuatorBase):
             a concrete subclass of ``PathwayBase``, e.g. ``LinearPathway``.
 
         """
-        self.force = force
-        self.pathway = pathway
-        super().__init__()
-
-    @property
-    def force(self) -> ExprType:
-        """The magnitude of the force produced by the actuator."""
-        return self._force
-
-    @force.setter
-    def force(self, force: ExprType) -> None:
         if not isinstance(force, ExprType):
             msg = (
                 f'Value {repr(force)} passed to `force` was of type '
@@ -173,6 +162,14 @@ class ForceActuator(ActuatorBase):
             )
             raise TypeError(msg)
         self._force = force
+
+        self.pathway = pathway
+        super().__init__()
+
+    @property
+    def force(self) -> ExprType:
+        """The magnitude of the force produced by the actuator."""
+        return self._force
 
     @property
     def pathway(self) -> PathwayBase:

--- a/sympy/physics/mechanics/tests/test_actuator.py
+++ b/sympy/physics/mechanics/tests/test_actuator.py
@@ -141,7 +141,7 @@ class TestForceActuator:
 class TestLinearSpring:
 
     @pytest.fixture(autouse=True)
-    def _linear_pathway_fixture(self) -> None:
+    def _linear_spring_fixture(self) -> None:
         self.stiffness = Symbol('k')
         self.l = Symbol('l')
         self.pA = Point('pA')

